docker-compose.yml

```diff
@@ -10,17 +10,6 @@ services:
       HASURA_GRAPHQL_DATABASE_URL: postgres://visionzero:visionzero@postgis:5432/atd_vz_data
       HASURA_GRAPHQL_ENABLE_CONSOLE: "true"
       HASURA_GRAPHQL_CONSOLE_ASSETS_DIR: /srv/console-assets
-  db-tools:
```

Comment:
ok cool, so this PR re-maps the dependencies of this container to `postgis` which is now being used for the snapshotting part of the local stack functionality. 👍

makes sense to me! i can't think of another dependency on this service.